templates/javascript/azion-mcp-template-no-auth/info.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "Azion MCP Template No Auth",
+  "message": "Template for Azion MCP Server without authentication, using JavaScript. Ideal for quick starts and public endpoints.",
+  "preset": "javascript",
```

At the `azion.config` file it says `typescript`, I'm assuming there is the correct one, right?

```suggestion
  "preset": "typescript",
```